### PR TITLE
[Review] IAE-10428

### PR DESCRIPTION
**Pull request by madhaviambati-rei**

### CODE REVIEW

File: src/ui-kit/form-controls/upload-v2/upload-v2.template.html [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+           [showClose]="true"
+           [closeOnOutsideClick]="true"
+           [closeOnEscape]="true"
+           [type]="'warning'"
+           [title]="uploadFileActionModalConfig.title"
+           [submitButtonLabel]="'Ok'"
+           [cancelButtonLabel]="'Cancel'"
+           (onSubmit)="onActionModalSubmit($event)"
+           (close)="onUploadModalCloseChange($event)">
+  <ng-content select=".upload-v2-action-modal"></ng-content>
```
Comment (diego-ruiz) [inline src/ui-kit/form-controls/upload-v2/upload-v2.template.html]: the `upload-v2` is alittle redundant, can you use `.action-modal-content`?

File: src/ui-kit/types.ts [hunk 2]
```diff
@@ -344,3 +344,32 @@ export interface UploadedFileData {
...
   icon: any;
 }
+
+export interface UploadFileActionModalConfig {
+    /**
+     * sets the upload modal title
+     */
+    title?: string,
+    /**
+     * sets the upload modal description
```
Comment (diego-ruiz) [inline src/ui-kit/types.ts]: the description is no longer being used, can we remove it in the meantime?

Comment (madhaviambati-rei) [reply src/ui-kit/types.ts]: I was using in ui-kit-demo.page

File: src/ui-kit/form-controls/upload-v2/upload-v2.template.html (hunk 1 above)
Comment (madhaviambati-rei) [reply src/ui-kit/form-controls/upload-v2/upload-v2.template.html]: Sure

**Pull request merged**